app/routes/docs/schema-browser/latest.$.ts

```diff
@@ -0,0 +1,19 @@
+import { Octokit } from '@octokit/rest'
+import type { DataFunctionArgs } from '@remix-run/node'
+import { redirect } from '@remix-run/node'
+
+const githubData = {
+  owner: 'nasa-gcn',
+  repo: 'gcn-schema',
+}
+const octokit = new Octokit()
```

Comment:
Reuse the existing octokit client instance in schema-data.ts.

That may mean defining a function in schema-data.ts that gets the latest release.